docs/api_reference/api_reference.py

```diff
@@ -413,10 +413,10 @@ def ex_alerts3():
         cls=AlertT.error)
 
 def ex_toasts1():
-    return Toast("First Example Toast", cls=(ToastHT.start, ToastVT.bottom))
+    return Toast("First Example Toast", cls=(ToastHT.start, ToastVT.bottom), dur=300)
```

Comment:
I set the duration to 5 minutes so that the toast is unlikely to disappear while the user is viewing the docs. This isn't foolproof but should hopefully do the job in 90%+ of cases.